pkg/util/kubernetes/kubelet/kubelet.go

```diff
@@ -53,6 +54,7 @@ type KubeUtil struct {
 	filter                 *containers.Filter
 	waitOnMissingContainer time.Duration
 	podUnmarshaller        *podUnmarshaller
+	podResources           *PodResourcesClient
```

Comment:
nit: We might want to name this as `podResourcesClient` (just as the `kubeletClient`).